### DISCUSSION TITLE
Case Sensitivity issue on the Request Logs

### DISCUSSION THREAD

**Discussion by mahirchavda** [pocketbase/pocketbase#1956] [Q&A] [answered]
The method field value is stored in the lower case in the backend but in the UI it is shown in upper case. So if I try to search `method="GET"`, it is not working.

Possible Solution:
* Store method value in the upper case or Show the value as it is (in lowercase) to remove confusion.

Also, Is it possible to show the collection name somehow to make logs more readable?

![image](https://user-images.githubusercontent.com/25617583/222353223-fb700d1a-dc85-4362-ab53-bd56ed2a1344.png)

**Comment by ganigeorgiev** [comment, marked as answer]
I'll consider updating the UI (or maybe convert the `method` values to uppercase).

> Is it possible to show the collection name somehow to make logs more readable?

At the moment this is not possible. If you want to filter by collection name/id, you can try searching the url with the `~` operator: `url ~ "myCollection"`.

The logs UI will change in the future since there is an idea to "generalize" the logs so that we can enable custom and non-request related logs (it is already in the [roadmap](https://github.com/orgs/pocketbase/projects/2), but for now remains a low priority).

**Reply by mahirchavda** [reply]
Thanks 👍
